src/overset/TiogaSTKIface.C

```diff
@@ -350,7 +350,7 @@ TiogaSTKIface::populate_overset_info()
   ThrowAssert(osetInfo.size() == 0);
 
   VectorFieldType *coords = meta_.get_field<VectorFieldType>
-    (stk::topology::NODE_RANK, realm.get_coordinates_name());
+    (stk::topology::NODE_RANK, coordsName_);
```

Comment:
There is a function meta_.coordinate_field(); stk-io calls meta_.set_coordinate_field();

Reply:
@alanw0 In Nalu, we use `coordinates` (the field name read from the Exodus file) when there is no mesh motion, but use `current_coordinates` when there is mesh motion. Previously, this field name was retrieved from `realm` within the TIOGA-STK interface. Now that I've removed Realm object in this interface, it is now being passed through the constructor. using `meta_.coordinate_field()` won't be appropriate as it would still return `coordinates` in case of mesh motion.